RecDP/pyrecdp/primitives/operations/rag/document_splitter.py

```diff
@@ -0,0 +1,70 @@
+from typing import Optional, Dict, List, Any
+
+from pyspark.pandas import DataFrame
+from ray.data import Dataset
+
+from pyrecdp.primitives.operations.base import BaseLLMOperation, LLMOPERATORS
+
+
+def split_text(text: str, text_splitter: str, text_splitter_kwargs: Optional[Dict] = None) -> List[str]:
+    text_splitter_kwargs = text_splitter_kwargs or {}
+    if text_splitter is None:
+        from langchain.text_splitter import RecursiveCharacterTextSplitter
+        text_splitter = RecursiveCharacterTextSplitter(**text_splitter_kwargs)
+        return text_splitter.split_text(text)
+    else:
+        try:
+            import importlib
+            module = importlib.import_module("langchain.text_splitter")
+            class_ = getattr(module, text_splitter)
+            text_splitter = class_(**text_splitter_kwargs)
+            return text_splitter.split_text(text)
+        except Exception:
+            raise NotImplementedError(f"text_spliiter {text_splitter} is not supported yet")
+
+
+class DocumentTextSplitter(BaseLLMOperation):
```

Comment:
@chaojun-zhang , can you check the difference of this one and the documentsplit you implemented before? Can we unify them as one operator?
Or based on our previous understanding, we need to align on the strategy of shall we convert langchain function one by one?

Reply:
Yes. updated.